### ISSUE TITLE
Enhancement: Whitelist hosting providers for security headers.

### ISSUE THREAD

**Issue by simbleau** [Jieiku/abridge#48]
Off the top of my head, if someone is using the security headers, chances are they can't enable them because they don't control the web server. As a result, I think it would be apparently beneficial to whitelist the common free hosting services:
- netlify
- github pages
- cloudfare pages
- gitlab pages

**Comment by Jieiku** [comment]
yeah, I will do some work on this, rather than whitelist them all I will make some config switches.

**Comment by Jieiku** [comment]
I think I am going to put some sensible defaults like you suggest, and I am going to put the entire thing into an html string in the config.toml, its one of those values with a million settings, so this will be easier for users to modify it as they need it.

**Comment by Jieiku** [comment]
These values are now configurable. If you have any changes you want to make to the default value of config.extra.security_header_csp just send a pull request and I will merge it.
